controller/gui/style.css

```diff
@@ -95,12 +100,24 @@
 
 .d-Network__Form {
     display: flex;
-    margin-bottom: 2rem;
+    flex-direction: column;
+    margin: 1rem 0;
+}
+
+.d-Network__Label {
+    width: 50%;
+    padding-bottom: 0.5rem;
+    font-size: 1.2rem;
 }
 
 .d-Network__Input {
     width: 50%;
-    margin-right: 1rem;
+    margin-bottom: 1rem;
+}
+
+.d-Network__Button {
+    width: 50%;
+    margin-top: 1rem;
```

Comment:
Is this outcome intended?

![image](https://user-images.githubusercontent.com/47458/98004225-f46b6400-1def-11eb-9b52-b73a4c531f15.png)

Reply:
Well, maybe better to reverted this: https://github.com/dividat/playos/pull/44/commits/ac6e4f1788c69cac311ac1c3f0fc5b006536cdfb.